CWE/events/italy.txt

```diff
@@ -301,10 +422,22 @@ country_event = {
 
 	option = {
 		name = "Enact the new laws!"
-		relation = { who = USA value = -100 }
-		relation = { who = ENG value = -100 }
-		relation = { who = FRA value = -100 }
-		relation = { who = RUS value = 50 }
+		random_country = {
+                       limit = { tag = USA government = democracy is_vassal = no }
+                       relation = { who = THIS value = -100 }
+        }
+		random_country = {
+                       limit = { tag = ENG government = democracy is_vassal = no }
```

Comment:
The check for ENG should be government = hms_government since they are a constitutional monarchy